docs/application/dotnet/guides/multimedia/audio.md

```diff
@@ -140,8 +140,34 @@ To query audio device information:
         > **Note**   
 		> The initial running state of the connected device is `false`, which means the connected device is not running.
 
+-   More functionality for AudioDeviceType.UsbAudio and AudioDeviceIoDirection.Output device, you can utilize more functionality with the following methods of the [Tizen.Multimedia.AudioDevice](https://samsung.github.io/TizenFX/latest/api/Tizen.Multimedia.AudioDevice.html) class:
+    -   To get the supported sample formats and to set the specific sample format:
 
+        1.  Retrieve the list of the currently supported sample formats with the `GetSupportedSampleFormats()` method:
 
+            ```
+            IEnumerable<AudioSampleFormat> supportedFormats = usbOutputDevice.GetSupportedSampleFormats();
+            ```
+
+        2.  Choose one of the supported sample formats, then set it with the `SetSampleFormat()` method:
```

Comment:
@mijin85cho Please verify the following suggestion with respect to removing 'method' is fine?

Select one of the supported sample formats, then set it with `SetSampleFormat()`:

Reply:
> @rishu-samsung Please keep the method.
> 
> Select one of the supported sample formats, then set it with the `SetSampleFormat()` method:
> 
> Question. Is there any specific reason that you changed "Choose" to "Select"?
> As I don't see the reason of it, would you tell me why?

@mijin85cho Sure, going forward will retain the word "method".

I agree to your point, although as per Google Style Guide, "Select" is more recommended to select an item from multiple options. For more information, see: https://developers.google.com/style/word-list#letter-s

Please let us know your thoughts regarding the same.

Reply:
@rishu-samsung If so, I accept.